src/apis/user.rs

```diff
@@ -18,6 +18,7 @@ pub fn apis(
     new_user(db_manager.clone())
         .or(login(db_manager.clone()))
         .or(change_password(db_manager))
+        .or(me())
```

Comment:
I think this route should be gated by the openid feature: we want to deactivate this route when openid is activated (and all password-based authentication, really) because when I implemented openid support I had to keep inserting tokens in the user database, but they are all bogus